### PR TITLE
refactored dataloaders

### PR DESCRIPTION
**Pull request by crewtool**

### CODE REVIEW

File: nano/model.py [hunk 1]
```diff
@@ -133,7 +132,7 @@ def run(cfg):
         metric_logger.run["job_config"] = cfg
         upload_config_file(metric_logger)
 
-    torch.manual_seed(cfg.training.seed)
+    torch.manual_seed(cfg.trainer_factory.train_dataloader.seed)
```
Comment (j321m) [inline nano/model.py:135]: maybe we schould have separate torch seed argument, and in yaml eval it as dataloader seed?

Comment (j321m) [reply nano/model.py:135]: for now ok.

**Review by j321m — APPROVED** [review]
ok, works, aceeeept
